### DISCUSSION TITLE
Image got decreased quality, how could I post blog with original quality

### DISCUSSION THREAD

**Discussion by longtth** [forem/forem#19931] [Q&A] [answered]
I often need to embed some images to my blogs, especially when writting guideline-blogs but when I check the blog output, I see that images have been deceased, sometimes, my reader can not read the content on those images. 
eg: https://dev.to/longtth/code-block-trong-msword-bp8

Is this intentional or just a bug? 
How could I post blog on dev.to with image quality at about 1920x1080 resolution? 

I expect that those images to keep quality (see the original image in new tab when clicked) like I post on github like below 

![image](https://github.com/forem/forem/assets/1010849/2391a9f9-fc6d-4b9e-8078-319af8be0017)

**Comment by oftharsh** [comment, marked as answer]
If you like it please mark it as answer.

It seems like the issue you're facing might be related to the way images are being displayed or compressed on the DEV Community platform. DEV.to does have some automatic image compression and resizing features to optimize page loading speed. However, if the images are not displaying properly or losing quality, it could be due to a few reasons:

1. **Compression Settings:** The platform might be compressing images to reduce file sizes, which could impact image quality.

2. **Responsive Design:** DEV.to might be optimizing images for responsive design, which could lead to variations in image dimensions.

3. **Markdown Formatting:** Ensure that you're using correct Markdown formatting for images. The Markdown syntax for embedding images on DEV.to is `![Alt Text](Image URL)`.

4. **Image Resolution:** While the platform might resize images, using a resolution close to 1920x1080 could improve image quality.

5. **Image Hosting:** Consider hosting images on an external service like Imgur or GitHub and using the direct URL in your Markdown.

6. **Checking on Different Devices:** Sometimes image quality might look different on different devices or browsers. Make sure to check on multiple devices to confirm the issue.

If you're not seeing the expected image quality on DEV.to, you might want to reach out to their support or community forums to get specific assistance for this issue. When posting blogs with images, you can try uploading images with a resolution close to 1920x1080 and ensure that the image URLs are correct in your Markdown content.

Remember, while high-resolution images can enhance the visual quality of your blogs, they might also impact the page loading speed, so finding a balance between quality and performance is important.
